docs/reference/search-application/apis/search-application-render-query.asciidoc

```diff
@@ -57,21 +70,16 @@ PUT _application/search_application/my-app
         "query": {
           "multi_match": {
             "query": "{{query_string}}",
-            "fields": [{{#text_fields}}"{{name}}^{{boost}}",{{/text_fields}}]
+            "fields": ["title^{{title_boost}}", "description"]
```

Comment:
As discussed offline, I believe it's OK to provide a simple example, but I don't see why we can't keep a list example for people to refer to as well? I do not believe this should be blocked by other behavior in templates outside of search applications because it educates people how to use templates effectively for search applications.

Reply:
I agree that we should have a list example as well. Perhaps we link to this example from the enterprise search docs? https://www.elastic.co/guide/en/enterprise-search/current/search-applications-templates.html#search-applications-templates-bm25-template

Reply:
FYI @Mikep86 the Search Applications docs have moved fully into the Elasticsearch docs as of 8.11.

See https://www.elastic.co/guide/en/elasticsearch/reference/8.11/search-application-overview.html.